quickwit-index-config/src/default_index_config/field_mapping_type.rs

```diff
@@ -32,6 +32,8 @@ pub enum FieldMappingType {
     Text(TextOptions, Cardinality),
     /// Signed 64-bits integers mapping type configuration.
     I64(IntOptions, Cardinality),
+    /// Unsigned 64-bits integers mapping type configuration.
```

Comment:
FYI, for next time: `64-bit` ([hyphen in compound adjective with numbers](https://www.grammarly.com/blog/hyphen-in-compound-adjective-with-numbers/)).